Qlik.Sense.RestClient/RestClient.cs

```diff
@@ -143,8 +143,12 @@ private static async Task<string> LogReceive(Task<string> messageTask)
             if (!Directory.Exists(path)) throw new DirectoryNotFoundException(path);
             if (!File.Exists(clientCertPath)) throw new FileNotFoundException(clientCertPath);
             var certificate = certificatePassword == null
-                ? new X509Certificate2(clientCertPath)
-                : new X509Certificate2(clientCertPath, certificatePassword);
+                //This is a fix for when using asp.net on some windows servers, where the X509Certificate2constructor
+                //tries to access the private key store for local user, even for .pfx file, and if the profile is not
+                //loaded in IIS (even some cases when it is loaded) throws an error. This tells the constructor
+                //to look at the Local Computer key store.
+                ? new X509Certificate2(clientCertPath, "", X509KeyStorageFlags.MachineKeySet)

```

Comment:
Perhaps we should make the storage flag value a parameter instead? I suspect there will be cases where people do not want to use the MachineKeySet value. I think I would prefer an overloaded function similar to the one that exists for the X509Certificate2 constructor. Like this:
`LoadCertificateFromDirectory(string path, SecureString certificatePassword, X509KeyStorageFlags keyStorageFlags)`

Reply:
yes, that would provide flexibility. Sorry I have not been able to review this for a while, would you mind looking at it if you have a the time? I won't have free time till end of May now, schedule is rather busy.